components/Schedule/Schedule.tsx

```diff
@@ -87,20 +104,59 @@ export default function Schedule() {
   )
 }
 
+const Header = styled.section`
+  margin-top: 3rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+
+  h2 {
+    color: ${({ theme }) => theme.text.primary};
+  }
+`
 const TableContainer = styled.table`
-  margin: 5rem auto;
+  margin: 0 auto;
+  min-height: 800px;
+  width: 95%;
 `
 
 const TableHeader = styled.thead`
   height: 40px;
-  background-color: lightblue;
+  background-color: ${({ theme }) => theme.bg.secondary};
+  color: ${({ theme }) => theme.text.primary};
+  text-transform: capitalize;
 `
 const TableRow = styled.tr`
-  height: 1350px;
+  height: 80%;
 `
 const TableField = styled.td`
   height: 100px;
   width: 200px;
-  background-color: lavender;
+  background-color: ${({ theme }) => theme.bg.quarternary};
   vertical-align: top;
 `
+
+const SmallButton = styled.button`
+  border-radius: 5px;
+  background: ${({ theme }) => theme.bg.secondary};
+  padding: 1rem;
+  color: ${({ theme }) => theme.text.primary};
+  width: 120px;
+  font-size: 1.25rem;
+  outline: none;
```

Comment:
Why do you have this? I think it's best to remove. In a different comment where you also do this there is more context on why